packages/paste-core/components/chat-composer/src/PlaceholderWrapper.tsx

```diff
@@ -17,7 +17,7 @@ export const PlaceholderWrapper: React.FC<
     right="space40"
     display="flex"
     alignItems="center"
-    color="colorTextWeak"
+    color="inherit"
```

Comment:
question: Why this change?

Reply:
the color will be inherited from `ChatComposerContainer`. Otherwise in dark mode for both normal and disabled variant, the text color looks same. Inheriting from parent makes the disabled text a little dim

Reply:
fixed it, using the context approach to fix color for placeholder